src/main/kotlin/com/cognifide/gradle/aem/instance/InstancePlugin.kt

```diff
@@ -47,14 +46,13 @@ class InstancePlugin : AemPlugin() {
                 dependsOn(InstanceDown.NAME, InstanceUp.NAME)
             }
             register<InstanceCreate>(InstanceCreate.NAME) {
-                dependsOn(Resolve.NAME)
-                mustRunAfter(InstanceDestroy.NAME)
+                mustRunAfter(InstanceDestroy.NAME, InstanceResolve.NAME)
             }
             register<InstanceDestroy>(InstanceDestroy.NAME) {
                 dependsOn(InstanceDown.NAME)
             }
             register<InstanceSatisfy>(InstanceSatisfy.NAME) {
-                dependsOn(Resolve.NAME).mustRunAfter(InstanceCreate.NAME, InstanceUp.NAME)
+                mustRunAfter(InstanceResolve.NAME, InstanceCreate.NAME, InstanceUp.NAME)
```

Comment:
removing dependson will no longer affect instance up and resolving files unnecessarily